packages/edit-site/src/components/url-query-controller/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { getQueryArg, addQueryArgs, removeQueryArgs } from '@wordpress/url';
+
+export default function PostRouter() {
```

Comment:
```suggestion
export default function URLQueryController() {
```

Reply:
Whoops!  I even thought to myself that I needed to update that and then totally forgot.  🤣   thanks for fixing that!